pom.xml

```diff
@@ -40,4 +41,57 @@
 			</snapshots>
 		</repository>
 	</repositories>
+	
+	<profiles>
+		<profile>
+			<id>quality</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
```

Comment:
Removed default activation. 
I've tried this build on my laptop. Everything is fine. About the second question we need to be here so that sonar could pick up the defined properties and know where to take generated jacoco files for the coverage reports.